src/components/menuAppBar/NavButton.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./NavButton.scss";
+import { useRouter } from "next/router";
+
+type navButton = {
+  name: string;
+  route: string;
+  activeIcon: string;
+  inActiveIcon: string;
+};
+
+const NavButton = (props: navButton) => {
+  const router = useRouter();
+  const { name, route, activeIcon, inActiveIcon } = props;
+  const isActive = router.pathname === route ? true : false;
```

Comment:
this ? true :false is redundant. 
 === return always boolean.

Reply:
Ok, I've removed this redundant code.